R/onLoad.R

```diff
@@ -58,32 +58,35 @@
   sharedPaths <- tryCatch({
```

Comment:
The POI release notes mention some other dependencies such as PDFBox graphics2d but I guess we won't need those for the functionality exposed through XLConnect.

Reply:
That seems to be the case, unless we have some functionality that would use it but is not tested ? As far as I know XLConnect doesn't have the ability to generate PDFs.